examples/BLIS.rs

```diff
@@ -0,0 +1,191 @@
+        .subschedule(&[0], &|move_a| {
+            move_a
+                .tile_out(&[1, 1], false)
+                .place(KernelType::ValueAssign)
+        })
+        .subschedule(&[1], &|matmul_b| {
+            matmul_b.move_param(1, CpuMemoryLevel::RF, row_major(2), None)
+        })
+        .subschedule(&[1, 0], &|move_ba| {
+            move_ba
+                .tile_out(&[1, 1], false)
+                .place(KernelType::ValueAssign)
+        })
+        .subschedule(&[1, 1], &|matmul_bb| {
+            matmul_bb.move_param(2, CpuMemoryLevel::RF, row_major(2), None)
+        })
+        .subschedule(&[1, 1, 0], &|s| s.place(KernelType::ValueAssign))
+        .subschedule(&[1, 1, 1], &|s| s.place(KernelType::Mult))
+        .subschedule(&[1, 1, 2], &|s| s.place(KernelType::ValueAssign));
+    print_header("Impl resulting from manual scheduling");
+    pprint(&imp, PrintMode::Compact);
+    println!();
+    // alloc aa: (64×64, u8, L1) <- #0
+    //   alloc ab: (64×64, u8, L1) <- #1
+    //     alloc ac: (64×64, u8, L1) <- #2
+    //       tile (ad: (4×64, u8, L1) <-[0, 2]- aa, ae: (64×4, u8, L1, c1, ua) <-[3, 1]- ab, af: (4×4, u8, L1, c1, ua) <-[0, 1]- ac)
+    //         alloc ag: (64×4, u8, L1, c1, ua) <- ae
+    //           tile (ah: (4×4, u8, L1, c1, ua) <-[0, 1]- ad, ai: (4×4, u8, L1, c1, ua) <-[1, 2]- ag)
+    //             alloc aj: (4×4, u8, L1, c1, ua) <- ah
+    //               tile (ak: (2×4, u8, L1, c1, ua) <-[0, 2]- aj, al: (4×2, u8, L1, c1, ua) <-[3, 1]- ai, am: (2×2, u8, L1, c1, ua) <-[0, 1]- af)
+    //                 tile (an: (1×4, u8, L1, ua) <-[0, 2]- ak, ao: (4×1, u8, L1, c1, ua) <-[3, 1]- al, ap: (1×1, u8, L1, ua) <-[0, 1]- am)
+    //                   alloc aq: (1×4, u8, RF)
+    //                     tile (ar: (1×1, u8, L1, ua) <-[0, 1]- an, as: (1×1, u8, RF, ua) <-[0, 1]- aq)
+    //                       ValueAssign(ar, as)
+    //                     alloc at: (4×1, u8, RF)
+    //                       tile (au: (1×1, u8, L1, ua) <-[0, 1]- ao, av: (1×1, u8, RF, ua) <-[0, 1]- at)
+    //                         ValueAssign(au, av)
+    //                       alloc aw: (1×1, u8, RF)
+    //                         ValueAssign(ap, aw)
+    //                         Mult(aq, at, aw)
+    //                         ValueAssign(aw, ap)
+
+    print_header("The above Impl lowered to C");
+    imp.emit(&mut ToWriteFmt(io::stdout())).unwrap();
+    // #include <inttypes.h>
+    // #include <stdlib.h>
+    // #include <stdint.h>
+    // #include <stdio.h>
+    // #include <string.h>
+    // #include <time.h>
+    // #include <arm_neon.h>
+    //
+    // __attribute__((noinline))
+    // void kernel(
+    //   uint8_t *restrict aa,
+    //   uint8_t *restrict ab,
+    //   uint8_t *restrict ac
+    // ) {
+    //   for (int ad = 0; ad < 16; ad++) {
+    //   for (int ae = 0; ae < 16; ae++) {
+    //     TODO: supposed to have let Bc = B.pack_into() here?
```

Comment:
In my BLIS implementation and scheduled impl, we have packing here. But the generated code doesn't. I would like to know if this comes from just my misunderstanding, or if the implementation needs modification.

Reply:
For packing, I tried `.move_param(1, CpuMemoryLevel::RF, row_major(2), None)` but didn't work.

```console
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', /tmp/samkaufman/morello/src/scheduling_sugar.rs:72:10
```

Reply:
For more details, please read my weekly notes.